client/src/components/OverviewComponent/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+
+import './style.css';
+
+const OverviewComponent = ({
+  firstTitle,
+  secondTitle,
+  content,
+  buttonText,
+  imageSource,
+  btnLink,
+  isRight,
+}) => {
+  return (
+    <div className="overview-container">
+      {isRight ? (
```

Comment:
@AlaaSaadeddin You could replace this ternary with a boolean:
```
{isRight && (
  <div className="container-photo">
    <img src={imageSource} alt="Home />
  </div>
)}
```
and then you don't need the `null` at the end.